server.js

```diff
@@ -6,6 +6,7 @@ require('ejs');
 
 // Application Dependencies
 const cors = require('cors');
+const { request, response } = require('express');
```

Comment:
Does this do anything or cause issues? I've not seen expressed used this way before and since the req/res objects are passed in to the route this shouldn't be affected.

If you start getting errors on the server I'd look at this.